backend/x86_proc.mli

```diff
@@ -88,7 +88,22 @@ val windows:bool
 (** Whether calls need to go via the PLT. *)
 val use_plt : bool
 
+module SectionName : sig
+  type t 
+  val equal : t -> t -> bool
+  val hash : t -> int
+  val compare : t -> t -> int
+  val make : string list -> string option -> string list -> t
+  val from_name : string -> t
+  val name : t -> string
```

Comment:
I'd rename `name` to `to_string` and `from_name` to `of_string` to match conventions elsewhere.